### DISCUSSION TITLE
FileNotFoundError: [Errno 2] No such file or directory: 'D:\\comfyui\\ComfyUI\\custom_nodes\\BrushNet-main\\__init__.py'  Cannot import D:\comfyui\ComfyUI\custom_nodes\BrushNet-main module for custom nodes: [Errno 2] No such file or directory: 'D:\\comfyui\\ComfyUI\\custom_nodes\\BrushNet-main\\__init__.py'

### DISCUSSION THREAD

**Discussion by lindongyue7** [huggingface/diffusers#9351] [Q&A]
FileNotFoundError: [Errno 2] No such file or directory: 'D:\\comfyui\\ComfyUI\\custom_nodes\\BrushNet-main\\__init__.py'

Cannot import D:\comfyui\ComfyUI\custom_nodes\BrushNet-main module for custom nodes: [Errno 2] No such file or directory: 'D:\\comfyui\\ComfyUI\\custom_nodes\\BrushNet-main\\__init__.py'

**Comment by asomoza** [comment]
This is a comfyui and a custom node error, so it has nothing to do with diffusers, please open the issue in the corresponding repo.

I will close this for now, if you encounter a true diffusers error feel free to open another issue.
